core/src/main/java/org/apache/carbondata/core/datastore/columnar/BlockIndexerStorageForShort.java

```diff
@@ -17,38 +17,25 @@
 
 package org.apache.carbondata.core.datastore.columnar;
 
-import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-
-import org.apache.carbondata.core.constants.CarbonCommonConstants;
-import org.apache.carbondata.core.util.ByteUtil;
 
+/**
+ * Only try applying RLE on rowId and data
+ */
 public class BlockIndexerStorageForShort extends BlockIndexerStorage<byte[][]> {
```

Comment:
Please change class name to a more understandable one

Reply:
done